Add concept page about sampling (#2434)
open-telemetry/opentelemetry.io: content/en/docs/concepts/sampling/index.md

---
title: Sampling
description: Learn about sampling, and the different sampling options available in OpenTelemetry.
weight: 
---

With distributed tracing, you observe requests as they move from one service to
another in a distributed system. It’s superbly practical for a number of
reasons, such as understanding your service connections and diagnosing latency

> issues, among many other benefits. If distributed tracing is a new topic for
> you, make sure to read this post on distributed tracing and sampling.

Reviewer: ```suggestion
issues, among many other benefits.
```

However, if the majority of all your requests are successful 200s and finish

> without latency or errors, do you really need all that data? Here’s the

Reviewer: ```suggestion
without unacceptable latency or errors, do you really need all that data? Here’s the
```

thing—you don’t always need a ton of data to find the right insights. _You just
need the right sampling of data._

![Illustration shows that not all data needs to be traced, and that a sample of data is sufficient.](traces_venn_diagram.png)

The idea behind sampling is to control the spans you send to your observability
backend, resulting in lower ingest costs. Different organizations will have
their own reasons for not just _why_ they want to sample, but also _what_ they
want to sample. You might want to customize your sampling strategy to:

> - **Manage costs**: You risk incurring heavy charges from the relevant cloud
>   provider or vendor if you’re exporting and storing all your spans.

Reviewer: ```suggestion
- **Manage costs**: If you have a high volume of telemetry, you risk
incurring heavy charges from a telemetry backend vendor or cloud
provider to export and store every span.
```

Reviewer: Recommend caging it this way because in my experience, the large majority of people actually don't have a high volume of data, and this really isn't a good reason for them to adopt sampling. But the bigger organizations with lots of money on the line do.
